core-services/image-mirroring/README.md

```diff
@@ -5,7 +5,30 @@ cluster registry. To publish them, they need to be mirrored to an external
 repository, like Quay. This is achieved by periodic Prow jobs. These jobs
 consume image mappings from the subdirectories of this directory. These image
 mapping files determine what should be mirrored where and are usually separated
-by version.
+by version. In a nutshell, the steps are
+
+* send the push credentials to DPTP
+* promote image to internal registry
+* define mappings of mirroring
+* rehearse and then merge the mirroring job
+
+## Promote the built image in CI
+
+As a prerequisite, specify `promotion` snippet in [ci-operator's config](https://github.com/openshift/ci-tools/blob/master/CONFIGURATION.md#promotion)
+to promote the image built in the CI process. _Note that the promoting image must NOT overwrite other images_.
+If the image is promoted to a new namespace, e.g., 
```

Comment:
We should expect people to check in YAML to this repository, not run `oc` commands

Reply:
Done.